src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -56,6 +56,8 @@ class SchemaForm extends React.Component {
       this.setState({ formContext: _.set('pageTitle', newProps.title, this.state.formContext) });
     } else if (!!newProps.reviewMode !== !!this.state.formContext.reviewMode) {
       this.setState(this.getEmptyState(newProps));
+    } else if (!!newProps.prefilled !== !!this.state.formContext.prefilled) {
```

Comment:
Heh, this might be a new record for the number of useful `!`s I've seen in an `if` statement. 😛

Curiosity, though: Doing `!!` will force it to a boolean, which means that if we get something really bizarre like `newProps.prefilled = 'false'` and `formContext.prefilled = true`, this `if`'s condition will be met when it shouldn't. That _shouldn't_ be a problem, but then, you're forcing a boolean because of defensive programming, right? Just in case we get a weird value?